src/visualization/types/SimpleTable/style.scss

```diff
@@ -7,6 +7,7 @@
   display: flex;
   flex-direction: column;
   user-select: text;
+  font-family: $cf-code-font;
```

Comment:
how about moving this up under `flex-direction` so that this block is still alphabetized